soc/arm/nxp_lpc/lpc55xxx/Kconfig.series

```diff
@@ -0,0 +1,14 @@
+# Kconfig - LPC LPC55S6X MCU line
+#
+# Copyright (c) 2017, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_SERIES_LPC55XXX
+	bool "LPC5500 Series Family MCU"
+	select HAS_MCUX
+	select SOC_FAMILY_LPC
+	select CPU_HAS_SYSTICK
```

Comment:
If the series only include CM33 variants, CPU_HAS_SYSTICK is not needed. Actually the option is now called CPU_CORTEX_M_HAS_SYSTICK